added webservices sample app

ayush-webonise opened the pull request.

Commit:
added webservices sample app

rakshith-webonise commented on web-services-sample-app/AppDelegate.swift:
```diff
@@ -0,0 +1,54 @@
+        self.window = UIWindow(frame: UIScreen.mainScreen().bounds)
+        let storyboard = UIStoryboard(name: "Main", bundle: nil)
+        let initialViewController = storyboard.instantiateViewControllerWithIdentifier("GetWeatherVC")
+        self.navigationController?.pushViewController(initialViewController, animated: false)
```
Use Storyboard to embed in UINavigationController

rakshith-webonise commented on web-services-sample-app/GetWeatherVC.swift:
```diff
@@ -0,0 +1,78 @@
+    /** This function checks for validations whether the user has added only whitespaces in the text field
+     * \returns Nothing
+     */
+    func isValid(isValidString: NSString) -> Bool {
```
1) Provide proper naming for functions, 'isValid' does not suggest much about what the method does.Same goes for the parameter name 'isValidString'
2) In the documentation use this format:
- paramter parameterName: description
- returns: Description
3) The method returns Bool as a return value.The Documentation reads returns Nothing.Update this

rakshith-webonise commented on web-services-sample-app/ShowWeatherVC.swift:
```diff
@@ -0,0 +1,114 @@
+     */
+    func webServicesRequest() -> Void {
+        let scriptUrl = CommonConstants.WebServices.URL
+        let urlWithParams = scriptUrl + "cityName=\(enteredCityName)&appid=\(CommonConstants.WebServices.APPID)"
```
Add cityName and appid as constants

rakshith-webonise commented on web-services-sample-app/GetWeatherVC.swift:
```diff
@@ -0,0 +1,78 @@
+        self.checkValidation()
+    }
+    
+        
```
Remove extra spaces,one line space is sufficient

Commit:
updated the assignment with asked modifications

rakshith-webonise commented on web-services-sample-app/AppDelegate.swift:
```diff
@@ -0,0 +1,54 @@
+//        let initialViewController = storyboard.instantiateViewControllerWithIdentifier("GetWeatherVC")
+//        self.navigationController?.pushViewController(initialViewController, animated: false)
+//        self.window!.rootViewController = navigationController
+//        self.window?.makeKeyAndVisible()
```
Remove this code.

rakshith-webonise commented on web-services-sample-app/CommonConstants.swift:
```diff
@@ -0,0 +1,34 @@
+    struct StoryboardID {
+        static let SHOW_WEATHER_STORYBOARD_ID = "ShowWeatherVC"
+    }
+    struct AlertField {
```
provide a different name, AlertField sounds vague

rakshith-webonise commented on web-services-sample-app/GetWeatherVC.swift:
```diff
@@ -0,0 +1,79 @@
+        // Do any additional setup after loading the view, typically from a nib.
+    }
+
+    /** This function checks for validations whether the user has added only whitespaces in the text field
```
The format for documentation is not right, refer the previous comments for the right format

rakshith-webonise commented on web-services-sample-app/ShowWeatherVC.swift:
```diff
@@ -0,0 +1,114 @@
+    }
+    
+    /** This function populates the textfields and displays information each text field is populated to display the required information
+     * \param None
```
If no parameters , dont write the parameter list as Nothing , omit it all together.
If return is Void, mention as Void avoid using Nothing

Commit:
Code documentation arranged properly and removed commented code

rakshith-webonise commented on web-services-sample-app/ShowWeatherVC.swift:
```diff
@@ -0,0 +1,107 @@
+    }
+   
+    /// This function stores value of specific key in separate variables. The local variables are then converted to string to store them into model's object. The function also calls the displayWeatherDetails() function
+    ///- parameter: dict of NSDictionary type
```
Documentation format is not right, refer this and make corrections
/**
     Method returns the count for each sport
     - parameter section: denotes the sport index
     - returns:Int,the count of the books for that sport
     */

rakshith-webonise reviewed (commented):
Change the documentation format.Rest looks Good

Commit:
Modified Documentation

changes in the documentation format by replacing single line comments with multiple line comments
